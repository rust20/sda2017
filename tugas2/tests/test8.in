4 4 0 1 3 20
JUST MONIKA;1;1;panah
JuSt MoNiKa;1;1;panah
J sT M nK K;1;1;panah
jjjj jjjMon;1;1;panah
1;3;JUST MONIKA
3;3;JuSt MoNiKa
1;13;J sT M nK K
3;13;jjjj jjjMon
JUSTMONIKAJUST#ONIKA
##............######
JUST#ONIKAJUST#ONIKA

