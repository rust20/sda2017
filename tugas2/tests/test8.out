1,13 Pahlawan yang ikut:J sT M nK K
3,13 Pahlawan yang ikut:jjjj jjjMon
1,3 Pahlawan yang ikut:JUST MONIKA
3,3 Pahlawan yang ikut:JuSt MoNiKa
Akhir petualangan Gudako
Level Gudako: 1
Level pahlawan:
J sT M nK K: 1
jjjj jjjMon: 1
JUST MONIKA: 1
JuSt MoNiKa: 1
